bin/all_sky_search/pycbc_coinc_hdfinjfind

```diff
@@ -133,12 +137,17 @@ for trigger_file, injection_file in zip(args.trigger_files,
     time_sorting = time.argsort()
 
     logging.info('Read in the injection file')
-    indoc = ligolw_utils.load_filename(injection_file, False,
-                                       contenthandler=LIGOLWContentHandler)
-    sim_table = table.get_table(indoc, lsctables.SimInspiralTable.tableName)
-    inj_time = numpy.array(sim_table.get_column('geocent_end_time') +
-                           1e-9 * sim_table.get_column('geocent_end_time_ns'),
-                           dtype=numpy.float64)
+
+    if '.xml' or '.xml.gz' in injection_file:
+        indoc = ligolw_utils.load_filename(injection_file, False,
+                                           contenthandler=LIGOLWContentHandler)
+        sim_table = table.get_table(indoc, lsctables.SimInspiralTable.tableName)
+        inj_time = numpy.array(sim_table.get_column('geocent_end_time') +
+                               1e-9 * sim_table.get_column('geocent_end_time_ns'),
+                               dtype=numpy.float64)
+    else:
+        inj_file = h5py.File(injection_file)
```

Comment:
Probably a good idea to explicitly open the injection file in read-only mode.

Reply:
@titodalcanton Yep, fixed.